services/EUDAT_service_integration/B2SHARE_access_image/B2SHARE_access_image_builder.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+
+# This script builds the Docker image that enables access to a B2SHARE instance. It does so by copying scripts that call
+# the respective B2SHARE HTTP API functions into the image and by preparing the image for their invocation by installing the required dependencies.
+
+import sys
+
+try:
```

Comment:
One comment on style: it is not necessary to place imports in try/catch blocks. By default if an import fails an error is returned and the script exits, which is exactly what you also do.